### ISSUE TITLE
Missing Focus Indicators

### ISSUE THREAD

**Issue by jchow11** [bcgov/performance#741]
**Description**
Focus indicators allow keyboard only users to understand where they are on a page. They generally appear as a box outline around interactive elements (buttons, drop downs, tab navigation, etc) the user currently "has focus" on. The focus indicator moves upon clicking "Tab" or "Shift + tab" to move forward or backwards within the page. 

**User Story**
As a keyboard only user, I want a visual indicator when I navigate to each interactive element on a page, so that I know where I am currently active on the page.

**Acceptance Criteria (this ticket is "done" if)**
- Focus indicators exist for each interactive element on every page
- Focus indicators are highly visible and clear
- Focus indicators must match the accessibility standards (for colour contrast)

**Example video (watch with sound):**
https://www.loom.com/share/afd53a8c3554419eba30af88dfe77b43

**Comment by Abuchana** [comment]
@jchow11, are you able to provide more details for this ticket? Assignee, estimate, any labels that apply e.g., "bug"... Thank you!

**Comment by jchow11** [comment]
@Abuchana  I think this is a Travis question! The assignee will come from Travis and the estimate will come from the assignee.

**Comment by Abuchana** [comment]
@jchow11, thanks, I will ask Travis :-)

**Comment by steffipravasi** [comment]
Closing this ticket as I will be creating Accessibility tickets for individual page
